azure-proto-compute/VirtualMachineOperations.cs

```diff
@@ -286,5 +290,16 @@ public async Task<ArmOperation<VirtualMachine>> StartAddTagAsync(string key, str
                     return new VirtualMachine(ClientOptions, Resource as VirtualMachineData);
                 });
         }
+
+        private protected virtual VirtualMachine GetResource()
```

Comment:
Can we move this to ResourceOperationBase since all resource types will need this?